src/odemis/driver/orsay.py

```diff
@@ -2879,13 +2879,15 @@ def _referencedListener(self, parameter=None, attr_name="Actual"):
         Reads if the aperture plate is calibrated and updates the value of the referenced VA in axis X and Y.
         Gets called as callback by the Orsay server when the parameter changes value.
 
-        :param (Orsay Parameter) parameter: The parameter on the Orsay server that calls this callback
-        :param (str) attr_name: The name of the attribute of parameter which was changed
+        :param (Orsay Parameter) parameter: The parameter on the Orsay server that calls this callback. None is replaced
+        the Orsay parameter whenever it is called without this keyword argument.
```

Comment:
I don't fully understand this sentence: `None is replaced the Orsay parameter whenever it is called without this keyword argument.`
If I read the code correctly, when `parameter = None`, it is set to `self._hybridAperture.Calibrated`

Reply:
Is this better? To be hones an all in the other listeners we never documented this and it was assumed as required knowledge from the Orsay/Odemis system. But I don't think it can hurt to add something about it.
```
        :param (Orsay Parameter) parameter: The parameter on the Orsay server that calls this callback. When no Orsay 
        Parameter is provided the Orsay Parameter 'Calibrated' is retrieved from the Odemis side.
```

Reply:
sounds good